### DISCUSSION TITLE
Please give me back the "pull requests" button on the top nav.

### DISCUSSION THREAD

**Discussion by lztetreault-dev** [community/community#67142] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

With this now gone I have to use either the search or the side menu to go to my pull requests which adds extra clicks. This is annoying please give me back my buttons.

<img width="37" alt="image" src="https://github.com/community/community/assets/50848874/3b60ed0a-4cba-47f2-8b07-201b8f99a507">

The button use to live to the left of the notifications button

<img width="551" alt="image" src="https://github.com/community/community/assets/50848874/3eff1424-4c7b-420d-b425-06b692af629b">

**Comment by qalmurri** [comment]
it's possible?
